coral-schema/src/test/resources/base-complex-nullable-with-defaults.avsc

```diff
@@ -7,7 +7,7 @@
     "type" : "int"
   }, {
     "name" : "Int_Field",
-    "type" : [ "null", "int" ],
+    "type" : [ "int", "null" ],
```

Comment:
why do we still need these changes? Arent we surfacing whatever is the schema in the base avro/orc table?

Reply:
We need to change the order here because this is not a valid schema in Avro 1.10, and the table with this wrong schema can't be created.